contracts/mock/UniswapRouterMock.sol

```diff
@@ -0,0 +1,48 @@
+pragma solidity ^0.5.0;
+
+import "../external_interfaces/IUniswapV2Router01.sol";
+import "@openzeppelin/contracts/math/Math.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+
+contract UniswapRouterMock is IUniswapV2Router01 {
+    using SafeMath for uint256;
+
+    function addLiquidity(
+        address //tokenA
+        , address //tokenB
+        , uint amountADesired
+        , uint amountBDesired
+        , uint //amountAMin
+        , uint //amountBMin
+        , address //to
+        , uint //deadline
+    ) external returns (uint amountA, uint amountB, uint liquidity) {
+        return (amountADesired, amountBDesired, 1);
+    }
+
+    function swapExactTokensForTokens(
+        uint amountIn,
+        uint amountOutMin,
+        address[] calldata //path
+        , address //to
+        , uint //deadline
```

Comment:
Why mix the placement of commas? Why not just put them all at the end of the previous line?

Reply:
I thought it looked better this way and was easier to uncomment the lines, but a week later I think that was a terrible idea :)